src/Temporalio/Runtime.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+
+namespace Temporalio
+{
+    /// <summary>
+    /// Runtime for the Temporal SDK.
+    /// </summary>
+    /// <remarks>
+    /// This runtime carries the internal core engine and telemetry options for Temporal. All
+    /// connections/clients created using it, and any workers created from them, will be associated
+    /// with the runtime.
+    /// </remarks>
+    public sealed class Runtime
```

Comment:
Where are the runtime options supposed to be provided?

Reply:
They are not implemented yet. I may move this to a `Temporalio.Runtime` package so that those options can be provided.